tensorflow/core/util/tensor_bundle/tensor_bundle_test.cc

```diff
@@ -56,6 +61,14 @@ Tensor Constant_2x3(T v) {
   return Constant(v, TensorShape({2, 3}));
 }
 
+Tensor ByteSwap(Tensor t) {
+  Tensor ret = tensor::DeepCopy(t);
+  EXPECT_OK(ByteSwapTensor(&ret));
```

Comment:
Also, could you make this one TF_EXPECT_OK too?

Reply:
Sure, making that change now.